contracts/Trustcoin.sol

```diff
@@ -29,7 +29,7 @@ contract Trustcoin is OutgoingMigrationTokenInterface, ERC20TokenInterface, Safe
   address public migrationMaster; // The Ethereum address which is allowed to set the new token's address
 
   mapping (address => uint256) public balances; // (ERC20)
-  mapping (address => mapping (address => uint256)) public allowed; // (ERC20)
+  mapping (address => mapping (address => int256)) public allowed; // (ERC20)
```

Comment:
worth adding a comment here for readers what this value holds (as it can be negative etc.) - perhaps explain the whole shebang